packages/vite-config/src/vite-config.ts

```diff
@@ -17,18 +15,19 @@ export const config: UserConfig = {
     commonjsOptions: {
       transformMixedEsModules: true,
     },
-    rollupOptions: {
-      plugins: [rollupNodePolyFill()],
-    },
   },
   plugins: [
     react(),
     svgr({
       svgrOptions: {
         plugins: ['@svgr/plugin-jsx'],
         ref: true,
+        exportType: 'named',
```

Comment:
`named` allows to use named imports as a React component, e.g. `import { ReactComponent as Foo } from 'foo.svg'`